setup.py

```diff
@@ -152,12 +158,11 @@ def __init__(self, name, extra_cmake_args=[], libname=None):
 
 
 # The extension depends on these libs.
-# They're built along with the extension.
+# They're built along with the extension (unless using_system_libs() is True)
 AWS_LIBS = []
 if sys.platform != 'darwin' and sys.platform != 'win32':
-    if not using_system_libcrypto():
-        # aws-lc produces libcrypto.a
-        AWS_LIBS.append(AwsLib('aws-lc', libname='crypto'))
+    # aws-lc produces libcrypto.a
+    AWS_LIBS.append(AwsLib('aws-lc', libname='crypto'))
```

Comment:
I did this so that aws-lc is in `AWS_LIBS` even if we're not building it, in the same way that **all the other non-OS libs** are in `AWS_LIBS` even if we're not building them. This removes some weirdness later where we had to add `crypto` back into the list of libraries we need to link against